### ISSUE TITLE
OpenTX Companion log viewer chooses incorrect icon path in google earth

### ISSUE THREAD

**Issue by Jmelz** [opentx/opentx#5080]
file://C:/Users/user_name/AppData/Local/Temp/15160-track0.png

Since the path has file:// appended to it, it causes google earth to display a red X as an icon instead of the more common arrow icon.

**Comment by projectkk2glider** [comment]
The problem https://stackoverflow.com/a/24327346:
```
you could refer to the image by its absolute location (e.g. file:///C:/etnasss.jpg)
but Google Earth has security policy regarding access to local files on the file 
system outside the context of the KML file. You'd have to allow access to local 
files which generally is not recommended.
```

The solution is also described in the same answer.

**Comment by Jmelz** [comment]
I think the best solution is to use the built-in google earth icons, but I'm not sure how that is implemented. Sounds like the alternative is to include that image within the kmz file then.

I tested this, and it works (including the image in the kmz file and removing the file path, only referencing the file name in the kmz file). It's not necessarily ideal, but then you don't have to reference the google icons from the web either.

I called the file "track.png" and dropped it into the kmz file (first renamed to .zip, then dropped the file in). I edited the doc.kml file to reference simply "track.png" for the icon name. Yellow arrow shows up as expected. Another nice feature would be a setting in companion that allows one to choose 'relative to sea level' or 'relative to ground' for gps altitude so that this doesn't have to be changed each and every time in google earth.
